### PR TITLE
New package: shirowanisan.COEIROINK.GPU version 2.2.1

### PR DESCRIPTION
**Pull request by Exorcism0666**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/133921)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
+2.0GB

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
The nested installer isn't portable, but a self-extracting 7Zip file, which continues to run and extract after WinGet believes it's installed:

image

File: manifests/s/shirowanisan/COEIROINK/GPU/2.2.1/shirowanisan.COEIROINK.GPU.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.2.13 $debug=NVS1.CRLF.5-1-19041-3930.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: shirowanisan.COEIROINK.GPU
+PackageVersion: 2.2.1
+InstallerType: zip
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
```
Comment (stephengillie) [inline manifests/s/shirowanisan/COEIROINK/GPU/2.2.1/shirowanisan.COEIROINK.GPU.installer.yaml:9]:
```suggestion
  NestedInstallerType: exe
```

**Comment by stephengillie** [comment]
image

- https://github.com/microsoft/winget-pkgs/issues/107639

**Review by stephengillie — APPROVED** [review]

**Pull request merged**